src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -363,9 +363,15 @@ export class AppComponent implements OnInit, AfterViewChecked {
                         return new Promise<IGridDataRow[]>((resolve, reject) => {
                             self.dataService.getRows(offset, count, resultSet.batchId, resultSet.id).subscribe(rows => {
                                 let gridData: IGridDataRow[] = [];
-                                for (let i = 0; i < rows.rows.length; i++) {
+                                for (let row = 0; row < rows.rows.length; row++) {
```

Comment:
This is already being done at https://github.com/Microsoft/vscode-mssql/blob/ce0181c16464352c5804eb68a8d73e824bf131b3/src/views/htmlcontent/src/js/components/app.component.ts#L544 I'm confused, was something not working?